### PR TITLE
Homework8: Alina Shautsova

### PR DESCRIPTION
**Pull request by AlinaShautsova**

### CODE REVIEW

**Comment by Stml89** [comment]
Still can not see that PR title verification and auto-assign works =(((

File: homework8_subsequence.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        new_arr = arr[:i] + arr[i + 1:]
+        if test_for_increasing_function(new_arr):
+            return True
+    return False
+
+
+def answer(arr):
+    """Response output."""
+    if will_it_be_strictly_increasing(arr):
+        print("Answer : True")
```
Comment (Stml89) [inline homework8_subsequence.py]: please return True or Flase, instead of string

File: homework8_subsequence.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    else:
+        print("Answer : False")
+
+
+answer([1, 2, 3])
```
Comment (Stml89) [inline homework8_subsequence.py]: and here if you return bool type, so you can use assert here to check that your code works as expected

File: homework8_the_number_opposite.py
Comment (Stml89) [inline homework8_the_number_opposite.py:1]: pls, use function here

File: homework8_validate.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+"""Module providing a function printing python version."""
+number_of_credit_card = input("Enter your card number: ")
+while not number_of_credit_card.isdigit():
+    print("Errors. Number of credit card must consist of digits. ")
+    number_of_credit_card = input("Enter your correct card number: ")
```
Comment (Stml89) [inline homework8_validate.py]: the same code line#2 and #5

File: homework8_validate.py
Comment (Stml89) [inline homework8_validate.py:1]: use function here as well

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: homework8_subsequence.py (hunk 1 above)
Comment (AlinaShautsova) [reply homework8_subsequence.py]: Done

File: homework8_subsequence.py (hunk 2 above)
Comment (AlinaShautsova) [reply homework8_subsequence.py]: Done

File: homework8_the_number_opposite.py
Comment (AlinaShautsova) [reply homework8_the_number_opposite.py:1]: I used function already. Just name of function had name "answer" insted of "solution".

File: homework8_validate.py (hunk 3 above)
Comment (AlinaShautsova) [reply homework8_validate.py]: These lines had the same variable but different meanings: 
input("Enter your card number: ")        and 
input("Enter your correct card number: ").
This is can be possible, isn't it? 
But I changed it, as you had said

File: homework8_the_number_opposite.py
Comment (AlinaShautsova) [reply homework8_the_number_opposite.py:1]: Done, but my solution to this task was also correct, as I understood :)

File: homework8_validate.py
Comment (AlinaShautsova) [reply homework8_validate.py:1]: Done

**Review by Stml89 — APPROVED** [review]
Approved , please merge PR

**Pull request merged**